### PR TITLE
Feature #615 - Create Proposal Amend Threshold Quorum

### LINKED ISSUE #615: Create proposal: AmendElectionRuleThreshold/AmendElectionRuleQuorum

### PR DESCRIPTION
**Pull request by abefernan**
Closes #615.

### CODE REVIEW

File: packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx [hunk 1]
```diff
@@ -71,6 +73,44 @@ const ProposalForm = (): JSX.Element => {
 
         return { ...commonOptions, type: ProposalType.RemoveCommitteeMember, committee, address };
       }
+      case ProposalType.AmendElectionRuleThreshold: {
+        const targetElectionRuleId = parseInt(
+          values[COMMITTEE_THRESHOLD_FIELD].substring(0, values[COMMITTEE_THRESHOLD_FIELD].indexOf(":")),
+          10,
+        );
```
Comment (webmaster128) [inline packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx]: Use helper as I commented in other PR

File: packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx [hunk 2]
```diff
@@ -71,6 +73,44 @@ const ProposalForm = (): JSX.Element => {
...
+          type: ProposalType.AmendElectionRuleThreshold,
+          targetElectionRuleId,
+          threshold,
+        };
+      }
+      case ProposalType.AmendElectionRuleQuorum: {
+        const targetElectionRuleId = parseInt(
+          values[COMMITTEE_QUORUM_FIELD].substring(0, values[COMMITTEE_QUORUM_FIELD].indexOf(":")),
+          10,
+        );
```
Comment (webmaster128) [inline packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx]: Use helper as I commented in other PR

File: packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx [hunk 3]
```diff
@@ -71,6 +73,44 @@ const ProposalForm = (): JSX.Element => {
...
+        const quorumArray = values[QUORUM_FIELD] && values[QUORUM_FIELD].split("/").map(n => parseInt(n, 10));
```
Comment (webmaster128) [inline packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx]: Do we have a field validator that checks the user input format?

Comment (abefernan) [reply packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx]: No, I was thinking of creating some tickets to extend [medulas' field validators](https://github.com/iov-one/ponferrada/blob/master/packages/medulas-react-components/src/utils/forms/validators/index.tsx) with new validators for the create proposal form and apply them in Governance

**Pull request merged**